README.md

```diff
@@ -131,7 +131,7 @@ Review the [FRAME runtime implementation](./runtime/src/lib.rs) included in this
 
 ### Pallets
 
-The runtime in this project is constructed using many FRAME pallets that ship with the [core Substrate repository](https://github.com/paritytech/substrate/tree/master/frame) and a template pallet that is [defined in the `pallets`](./pallets/template/src/lib.rs) directory.
+The runtime in this project is constructed using many FRAME pallets that ship with the [core Substrate repository](https://github.com/paritytech/substrate/tree/master/frame) and a template pallet that is [defined in the `pallets`](./pallets/living-assets-evolution/src/lib.rs) directory.
```

Comment:
```suggestion
The runtime in this project is constructed using many FRAME pallets that ship with the [core Substrate repository](https://github.com/paritytech/substrate/tree/master/frame) and a pallet that is [defined in the `pallets`](./pallets/living-assets-evolution/src/lib.rs) directory.
```

Reply:
I think generally this part of the doc should be removed, it doesn't add anything meaningful and doesn't belong to our project

Reply:
I agree ... I will create a PR to rewrite the README: https://github.com/freeverseio/laos-evolution-node/pull/75